pymaker/model.py

```diff
@@ -61,3 +61,16 @@ def __str__(self):
 
     def __repr__(self):
         return pformat(vars(self))
+
+
+class TokenConfig:
+    def __init__(self, data: dict):
+        assert (isinstance(data, dict))
+
+        self.tokens = [Token(name=key,
```

Comment:
So the data is coming in indexed by token, but seems like it gets flattened into a list here.  Perhaps `self.tokens` should be a `dict` for O(1) lookup?

Reply:
Yeah that would definitely be better. I was continuing to flatten the data to maintain compatibility with existing schema expectations in `inventory-service` so we could just use this as a drop in class. An indexed dict would require corresponding updates to inventory service